.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: ci
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    name: 🧪 Test
+
+    env:
+      FLUTTER_STORAGE_BASE_URL: https://download.shorebird.dev
```

Comment:
Huh.  Wild to see us explicitly set this.  I wonder if we would rather wrap/fork the flutter script instead?

Reply:
This seemed like the least invasive way since it will result in fewer merge conflicts over time but happy to change the script itself